core/spec/models/spree/adjustment_spec.rb

```diff
@@ -13,6 +13,32 @@
 
   let(:adjustment) { Spree::Adjustment.create!(label: 'Adjustment', adjustable: order, order: order, amount: 5) }
 
+  describe 'scopes' do
```

Comment:
@Mafi88 - I guess this won't be a good idea because in future we will add specs for new scopes which don't require these variables.